collections/Laud_Misc/MS_Laud_Misc_130.xml

```diff
@@ -37,52 +36,235 @@
+                        <idno type="part">MS. Laud Misc. 130 - endleaf (fol. 1)</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem xml:id="MS_Laud_Misc_130-endleaf-item1">
+                        <locus>(fol. 1r-v)</locus>
+                        <author key="person_100227669">Ambrose</author>
+                        <title key="work_478">Expositio in Ps. CXVIII </title><note>(fragment)</note>
+                        <incipit defective="true">uestigiis <add place="above">et uiam</add> ueram inuenire nequeamus</incipit>
+                        <explicit defective="true">ut luceat omnibus qui in domo sunt, vt</explicit>
+                        <bibl>CSEL 62, pp. 300-303. Another fragment of this manuscript is MS. Laud Misc. 417 fol. 1.</bibl>
+                     </msItem>
+                  </msContents>
+                  <physDesc>
+                     <objectDesc form="codex">
+                        <supportDesc material="perg">
+                           <support>parchment</support>
+                        </supportDesc>
+                        <layoutDesc>
+                           <layout columns="2" writtenLines="29">Ruled with a hard point for two columns of 29 lines with double vertical bounding lines. Ruled space <dimensions type="ruled" unit="mm">
+                              <height>220</height>
+                              <width>175</width>
+                           </dimensions> (c. 180 mm. to the outer bounding lines)</layout>
+                        </layoutDesc>
+                     </objectDesc>
+                     <handDesc>
+                        <handNote script="minusculeCaroline">Caroline minuscule, 'jüngerer Lorscher Stil' according to Bischoff.</handNote>
+                     </handDesc>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate calendar="Gregorian" notAfter="0850" notBefore="0800">9th century,
+                           first half or second quarter</origDate> 
+                        <origPlace><country key="/place_7000084">Germany</country>, <settlement key="place_7012584">Lorsch</settlement>, <orgName key="org_143763943">Benedictine abbey</orgName></origPlace> (Bischoff)
+                     </origin>
+                     <provenance notBefore="0800"><orgName role="fmo"  key="org_143763943">Lorsch, Benedictine abbey</orgName>: probably  identifiable in the library catalogues (A. Häse, <title>Mittelalterliche Bücherverzeichnisse aus Kloster Lorsch. Einleitung, Edition und Kommentar</title> (2002), no. 213). </provenance>
+                  </history>
+               </msPart>
             </msDesc>
          </sourceDesc>
       </fileDesc>
       <revisionDesc>
+         <change when="2021-09">Description fully revised for digitization.</change>
```

Comment:
Missing name (unless you want to start leaving this out).

Reply:
I don't usually include it if it is already given in `recordHist/source` - but do you think we should?